examples/rln/build/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Better to include build folder itself in gitignore at higher level.

Reply:
I added this `.gitignore` inside the `build/` folder, so the folder itself is commited to the repo, and any file created within the folder is ignored.